ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -312,9 +312,9 @@ public EndpointResponse handleKsqlStatements(
       statements.forEach(s -> {
         if (s.getUnMaskedStatementText().toLowerCase().contains("terminate")
             || s.getUnMaskedStatementText().toLowerCase().contains("drop")) {
-          QueryLogger.info("Query terminated", s.getStatementText());
+          QueryLogger.info("Query terminated", s.getMaskedStatementText());
```

Comment:
Just noticed this, applies to earlier versions as well: there's a case to be made that the query logger should actually get the unmasked version, because it performs its own masking via the QueryAnonymizer, and the QueryAnonymizer has different logic for masking. For example, the QueryAnonymizer is type-aware, and masks string literals differently from numeric literals.

I'm fine with leaving this as is on these older branches but on master we need to work out details of how we want the two to interact. As currently written, we are changing the outputs of the QueryLogger by passing an already masked statement to the QueryAnonymizer. Also, the QueryAnonymizer is missing test coverage for INSERT VALUES and CREATE CONNECTOR statements, so we'll want to add that too. Given the expanded scope of this work, let's save this for master-only.